### PR TITLE
Issue #539: Remove unneeded theme functions from dblog module.

### LINKED ISSUE #539: [DX] Remove theme_dblog_message() since it outputs only a link
There's no reason to "theme" an anchor tag.  Let's just move the generation of this link to where this function was called and remove this theme function entirely. 

For reference here's what's going on in [Drupal 8](https://www.drupal.org/node/1898060)

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/539

### CODE REVIEW

File: core/modules/dblog/dblog.admin.inc [hunk 1]
```diff
@@ -62,13 +62,16 @@ function dblog_overview() {
     else {
       $user_info = theme('username', array('account' => $dblog));
     }
+
+    // here
+
```
Comment (quicksketch) [inline core/modules/dblog/dblog.admin.inc]: This should be removed. ;)

File: core/modules/dblog/dblog.admin.inc [hunk 2]
```diff
@@ -165,6 +168,9 @@ function dblog_event($id) {
       $user_label = t('User');
       $user_info = theme('username', array('account' => $dblog));
     }
+    
+    // here
+    
     $rows = array(
```
Comment (quicksketch) [inline core/modules/dblog/dblog.admin.inc:170]: And this. :)

**Comment by jenlampton** [comment]
hahaha whoopsie :)

**Comment by jenlampton** [comment]
Pushed a revised version.

**Pull request merged**